torch_geometric/utils/get_mp_out.py

```diff
@@ -0,0 +1,49 @@
+from typing import Callable, List
+
+import torch
+from torch import Tensor
+
+from .subgraph import get_num_hops
+
+
+def get_intermediate_messagepassing_embeddings(model: torch.nn.Module,
```

Comment:
```suggestion
def get_message_passing_embeddings(model: torch.nn.Module,
```
Would be in favor of making this function name a little bit smaller :)